lib/helpers.js

```diff
@@ -1,6 +1,21 @@
 const fs = require('fs');
 const path = require('path');
 
+/**
+ * @typedef {import('../typings').SvelteParserOptions} SvelteParserOptions
+ * @typedef {'fileContent' | 'filename' | 'encoding'} FileOptionKeys
+ * @typedef {{content?: string; template?: string, scripts?: unknown[], styles?: unknown[] }} FileStructure
```

Comment:
Why us a `unknown` are used for `scripts` and `styles` properties?

Reply:
The HtmlBlock type was created after and I forgot to update them.
I Will update them.